### ISSUE TITLE
Unsure about the "Opened UTC" column in default report

### ISSUE THREAD

**Issue by ngirard** [ad-si/TaskLite#49]
I don't understand the meaning of the "Opened UTC" column in the default report.

The documentation doesn't discuss it (for instance, it doesn't appear in the State/Field table [here](https://tasklite.org/concepts.html#states)).

**Comment by ngirard** [comment]
If I understand correctly, this corresponds to the timestamp extracted from the ULID, it that correct ?
Which means it cannot be modified, right ?
What happens when importing data from Taskwarrior ?

**Comment by ad-si** [comment]
Yeah exactly. It gets extracted from the ULID. When you import a task it automatically generates an ULID with the correct date. I mean, it can be changed by creating a new ULID, but all references of that ULID would need to be updated to. Currently there is no easy way to do that. I felt like the creation UTC should be immutable, and also it saves storage by combining the ID with the creation UTC. Maybe a command to change the creation date should be added (with a big warning that it changes the ID)
